### DISCUSSION TITLE
Bug: Document errors

### DISCUSSION THREAD

**Discussion by Codetotos** [gpbl/react-day-picker#1560] [General]
Bug encountered when using documents. If multiple dates are selected in the custom selection of multiple selection, clicking on the previously selected date will cancel the latest selected date

**Comment by Codetotos** [comment]
[img](https://s1.328888.xyz/2022/09/22/Ikrfk.jpg)

**Comment by Codetotos** [comment]
I am now formatting 'currentValue' and 'day' before 'day.splice' to use, looking forward to better writing

**Comment by gpbl** [comment]
@Codetotos I remember there was a bug with that example but I couldn't reproduce it anymore. Any way to reproduce it?

**Reply by Codetotos** [reply]
@gpbl Select multiple dates, then click on the date you originally selected

**Comment by gpbl** [comment]
Thanks for the report, this is fixed now: https://react-day-picker.js.org/basics/selecting-days#example-custom-multiple-select
